### PR TITLE
Feat/add paper and chip

### PR DESCRIPTION
**Pull request by chrisobdam**
@jeroenreijs Can you have a look at my chip color, doesn't seem to work

### CODE REVIEW

**Comment by jeroenreijs** [comment]
Since you are only using one override, you can use your custom className, in this case `chip`.
The root key can be removed from the styles object (in which the selector to the material ui class is not correct, '.&M' should be '')

But you can do it with just using this:

File: src/components/chip.js [hunk 1]
```diff
@@ -0,0 +1,38 @@
+(() => ({
+  name: 'Chip',
+  type: 'CONTENT_COMPONENT',
+  allowedTypes: [],
+  orientation: 'HORIZONTAL',
+  jsx: (() => {
+    const { Chip } = window.MaterialUI.Core;
+    const { label, disabled, variant } = options;
+    const isDev = B.env === 'dev';
+    const chip = (
```
Comment (jeroenreijs) [inline src/components/chip.js:10]: It is recommended to capitalize React component names

**Comment by vdberganton** [comment]
The outline option does not work and still shows the background
We don't have a proper icon yet for these components.

**Pull request merged**